aim/sdk/adapters/pytorch_lightning.py

```diff
@@ -35,17 +35,20 @@ class AimLogger(Logger):
     def __init__(self,
                  repo: Optional[str] = None,
                  experiment: Optional[str] = None,
+                 run_name: Optional[str] = None,
```

Comment:
Lets also move the argument to the back of the list. As the `AimLogger` allows both positional and keyword args, it might mess the order.